docs/source/whatsnew/1.0.2.txt

```diff
@@ -36,6 +36,12 @@ Enhancements
   produced a True on N or more days in the previous ``window_length``
   days (:issue:`1367`).
 
+- Remove risk-free rate from sharpe ratio calculation. (:issue:`853`).
```

Comment:
Seems like this fixed other issues with the sharpe ratio (mean versus cumulative returns, etc)?  Should we mention those and/or move this to Bug Fixes?